webapp/components/Modal/ReportModal.vue

```diff
@@ -8,14 +8,36 @@
 
     <!-- eslint-disable-next-line vue/no-v-html -->
     <p v-html="message" />
-
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-14T14:50:39Z" title="Monday, October 14th 2019, 4:50:39 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -50,7 +50,6 @@
 <script>
 import { SweetalertIcon } from 'vue-sweetalert-icons'
 import { reportMutation } from '~/graphql/Moderation.js'
-import { reportReasonCategoriesDatabaseList } from '~/../shared/moderation/report.js'
```


I think this didn't work in `jest` because of the how you are importing it @Tirokk 
why is there both a `~` and a `..`?? cause it lives in the root path? 
did you try `../../../shared/moderation/report.js`??

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-14T15:44:59Z" title="Monday, October 14th 2019, 5:44:59 pm +02:00">Oct 14, 2019</time>_

---


Yes, for a pity I already have found that out and removed it.
We need a solution for that, in my eyes !!! @mattwr18